### DISCUSSION TITLE
Trying to compare strings

### DISCUSSION THREAD

**Discussion by Zcehtro** [mockoon/mockoon#909] [Help]
I'm trying to compare strings in route params and in an object property.

This is my code attempt:

```text
{{setVar 'user_id' (urlParam 'user_id')}}
{{#each (dataRaw 'users' 'list')}}
{{#if (includes '(this.userLogin)' '(@user_id)')}}
{{#if (includes '(@user_id)' '(this.userLogin)')}}
{
  "userLanguage": "EN",
  "userId": {{this.userId}},
  "userLogin": "{{this.userLogin}}"
}
{{/if}}
{{/if}}
{{/each}}
```

I'm not having any luck.

I know that Handlebars doesn't support string comparison. However, according to what I've found, it does allow custom helpers to be registered. Does Mockoon support registering new helpers?

If registering helpers is not supported, how can I perform a string comparison?

**Comment by 255kb** [comment]
You need to remove the quotes around the `includes` params: 
```
{{#if (includes this.userLogin @user_id)}}
{{#if (includes @user_id this.userLogin)}}
```
Otherwise, the content of each param is considered as a string and the helper will search for 'this.userLogin'.
